### PR TITLE
Cli/indentation

### PR DESCRIPTION
**Pull request by mjgoodman321**
Modified the last line in the print_cli function in print_functions.c. That technically makes this a ui issue, but as there is no ui team nor ui tag, it is being solved as a cli issue.

### CODE REVIEW

**Review by ecoble — APPROVED** [review]
LGTM! Thanks for fixing this.

File: src/ui/src/print_functions.c [hunk 1]
```diff
@@ -197,7 +197,7 @@ void print_cli(chiventure_ctx_t *ctx, window_t *win)
         wscrl(win->w, y - height + 2);
         y = height - 2;
     }
-    mvwprintw(win->w, y, 2, "> ");
+    mvwprintw(win->w, y, 0, "  > ");
```
Comment (Beza20) [inline src/ui/src/print_functions.c:200]: smooth fix!

Comment (Beza20) [reply src/ui/src/print_functions.c:200]: The indentation looks good now!

**Review by Beza20 — APPROVED** [review]

**Comment by ecoble** [comment]
Issue Score: ✔️+

Comments:
Nice job with this bug fix! Be sure to include a closing statement for all PRs, even small ones like this.

**Pull request merged**